include/chainblocks.h

```diff
@@ -70,13 +70,6 @@ enum CBInlineBlocks : uint32_t {
   CoreRefTable,
   CoreUpdate,
   CoreSwap,
-  CoreTakeSeq,
```

Comment:
These are still useful if we want to literally inline them during the threaded execution loop.
Would avoid completely an indirect call and just simply inline all in a cache line - there are drawbacks though, I would do it for some type, like TakeSeq maybe, or float vectors